app/views/providers/address_selections/show.html.erb

```diff
@@ -14,17 +14,17 @@
     <%= govuk_form_group show_error_if: @form.errors[:lookup_id].present? do %>
       <%= govuk_fieldset_header page_title %>
 
-      <div class="govuk-label">
+      <p class="govuk-label">
         <%= t('generic.address.postcode') %>
         <p class='govuk-body govuk-!-font-weight-bold'>
-          <%= @form.postcode %>
+          <%= @form.postcode.insert(-4, " ") %>
```

Comment:
You're inserting a spce both here and in the controller; is there any way of combining the two?

Also in the controller you do `insert(-3), ' '`, and here it's `insert(-4, " "` - shouldn't the space be inserted in the same place in both?